cli/http_util.rs

```diff
@@ -60,6 +63,19 @@ fn resolve_uri_from_location(base_uri: &Uri, location: &str) -> Uri {
 
 #[cfg(test)]
 use crate::tokio_util;
+#[cfg(test)]
+use futures::future::FutureExt;
+#[cfg(test)]
+use hyper::Body;
```

Comment:
Can we use git based dependencies for tokio and hyper in Cargo.toml -- both have moved to `std::future`?

Reply:
We can, but it's a bit of a pain. I figured I would hold off on that for now.

Reply:
I am asking this because I moved one of my own project to futures 0.3 and kept tokio 0.1. It was awkward and verbose. Lots of `unit_error().boxed().compat()` and converting back to use `.await`. Then I moved to tokio 0.2. It was way cleaner and easier.

Reply:
SGTM